tslint.json

```diff
@@ -24,7 +24,8 @@
     ],
     "no-var-requires": false,
     "no-bitwise": false,
-    "member-access": [true, "no-public"]
+    "member-access": [true, "no-public"],
+    "no-big-function": false
```

Comment:
can we not have the ignore here, but only as an exception in that specific test/storybook instead?
like `// tslint:disable:no-big-function` ?

Reply:
done